src/unix/notbsd/linux/musl/mod.rs

```diff
@@ -0,0 +1,131 @@
+pub type clock_t = ::c_long;
+
+pub type blkcnt_t = i64;
+pub type fsblkcnt_t = u64;
+pub type fsfilcnt_t = u64;
+pub type ino_t = u64;
+pub type off_t = i64;
+pub type stat64 = ::stat;
```

Comment:
Up above it looked like `struct stat` and `struct stat64` had different definitions, was that just coincidental though?